### PR TITLE
use :dir as compound selector

### PR DESCRIPTION
**Pull request by valdrinkoshi**
Use compound selector on elements for RTL styling.

Can be merged as https://github.com/Polymer/polymer/pull/4970 is released in polymer v2.3.0

### CODE REVIEW

File: paper-dropdown-menu-light.html [hunk 1]
```diff
@@ -139,9 +139,7 @@
         @apply --paper-dropdown-menu-input;
       }
 
-      /* Keep :host-context() to make it work where :dir doesn't exist */
-      :host(:dir(rtl)) #input,
-      :host-context([dir="rtl"]) #input {
+      #input:dir(rtl) {
```
Comment (notwaldorf) [inline paper-dropdown-menu-light.html]: shouldn't this be `:host(:dir(rtl)) #input` ?

Comment (valdrinkoshi) [reply paper-dropdown-menu-light.html]: `:host` would be superfluous in this case, and as suggested in [this comment](https://github.com/Polymer/polymer/pull/4964#discussion_r153965363) what we want in this case is the compound selector which would handle both the case of `<input dir="rtl">` and `<html dir="rtl"> ..... <input>`

Comment (notwaldorf) [reply paper-dropdown-menu-light.html]: but the input is in _your_ shadow dom, so who's going to set `dir="rtl"` on it?

Comment (valdrinkoshi) [reply paper-dropdown-menu-light.html]: true, no-one will set it on the input. The compound selector tho will have lower specificity and  clearer to read. The end result is the same, right?

Comment (notwaldorf) [reply paper-dropdown-menu-light.html]: Talked to @azakus: I understand now.

**Pull request merged**